.github/workflows/test-windows.yml

```diff
@@ -40,6 +40,11 @@ jobs:
       - name: Build
         run: dotnet build -c Release --verbosity minimal
 
+      - name: Setup Testcontainers Cloud Client
+        uses: atomicjar/testcontainers-cloud-setup-action@v1
+        with:
+          token: ${{ secrets.TC_CLOUD_TOKEN }}
+
```

Comment:
I'm afraid this won't work for forked PRs. Shall this be split by enabling only when working on the upstream rather than on forked-PRs?

Reply:
That would be great @v1v 👍

Reply:
Cool, I'll code something. Then likely, I might need your help with skipping those tests for forked PRs.